offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/update/UpdateOfferViewModel.groovy

```diff
@@ -49,7 +49,7 @@ class UpdateOfferViewModel extends CreateOfferViewModel{
         super.customer = offer.customer
         super.customerAffiliation = offer.selectedCustomerAffiliation
         super.projectManager = offer.projectManager
-        super.experimentalDesign = offer.experimentalDesign
+        super.experimentalDesign = offer.experimentalDesign.isPresent() ? offer.experimentalDesign.get() : ""
```

Comment:
Nice and concise 👍 

You could also do it like that:
```suggestion
        super.experimentalDesign = offer.experimentalDesign.orElse("")
```

Reply:
Nice idea, didn't know about that. Thanks for introducing 🤓